Design4/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Contact Page</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="icon">
+        <img class="icon" src="./contact-form-icon.svg" />
+      </div>
+      <div class="main">
+        <div class="headings">
+          <div class="heading1"><p>Interested in our business pricing?</p></div>
```

Comment:
why have not used 'h1' if it is heading1..?? Or is it good to wrap each line of text in a div? why you have not used 'p' tag directly